packages/fetch-router/demos/node/data.ts

```diff
@@ -0,0 +1,44 @@
+export interface Post {
+  id: string
+  title: string
+  content: string
+  author: string
+  createdAt: Date
+}
+
+let posts: Post[] = [
+  {
+    id: '1',
+    title: 'Welcome to the Blog',
+    content: 'This is a simple blog demo built with fetch-router on Node.js.',
+    author: 'Admin',
+    createdAt: new Date('2025-01-01'),
+  },
+  {
+    id: '2',
+    title: 'Getting Started with fetch-router',
+    content: 'fetch-router is a minimal, composable router built on the web Fetch API.',
+    author: 'Admin',
+    createdAt: new Date('2025-01-02'),
+  },
+]
+
+export function getPosts() {
+  return [...posts].sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime())
```

Comment:
This can be simplified to using [`Array.prototype.toSorted()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted)


```suggestion
  return posts.toSorted((a, b) => b.createdAt.getTime() - a.createdAt.getTime())
```